docs/yul.rst

```diff
@@ -1095,6 +1095,7 @@ undefined behavior.
 
 - Control-flow should not jump into or out of verbatim blocks,
   but it can jump within the same verbatim block.
+- In particular, reverting or returning from the block is *not* allowed.
```

Comment:
Note that in https://github.com/ethereum/solidity/pull/15542#discussion_r1852823712 this was supposed to be merged with the bullet point above.

```suggestion
  In particular, reverting or returning from the block is *not* allowed.
```